5_databases/week3/New_Query_1708896606910.sql

```diff
@@ -0,0 +1,198 @@
+-- Active: 1707334336522@@localhost@3306@meal_sharing
+-- Create the Reservation table
+CREATE TABLE Reservation (
```

Comment:
Hey hey 

Perhaps here you could have NOT NULL in some of the values, to indicate that they can't be empty. 

Also having AUTO_INCREMENT in the ID (of this and other tables) will help you, so every time you add something to the table, the id gets automatically generated :)